docusaurus/docs/home.md

```diff
@@ -0,0 +1,14 @@
+# Developer Documentation
```

Comment:
The header `Docs` link still goes to `Concepts` instead of here

Reply:
Fixed